test/test_get.py

```diff
@@ -46,20 +72,35 @@ def test_download_file(downloader):
 
     # test caching
     paths = downloader.download(resource)
-    # should not download again
-    assert paths[0] is None
-
-    # manipulate cache dict to test expiration (datetime format)
-    downloader.cache_dict["test_resource"][
-        "date_downloaded"
-    ] = datetime.now() - timedelta(days=8)
+    # should not download again (assert that cache folder is returned)
+    if downloader == "downloader_without_specified_cache_dir":
```

Comment:
these conditionals both fail, because downloader is an instance of Downloader, not a string

Reply:
True. :see_no_evil: Learning: never trust what ChatGPT is generating